### ISSUE TITLE
cursor css not working

### ISSUE THREAD

**Issue by exoess** [Alexays/Waybar#2525]
i tried setting the css for hovering on workspaces to have the cursor become a pointer, but it spits out this error `[2023-09-21 18:49:22.932] [error] style.css:37:10'cursor' is not a valid property name`, not sure if this is a missing feature or an issue. thanks for any help you can give

**Comment by mutoroglin** [comment]
I do not think that this is a bug since `cursor` is not a supported [property of gtk3](https://docs.gtk.org/gtk3/css-properties.html)

**Comment by exoess** [comment]
is there a way to change the it in gtk css?

**Comment by Alexays** [comment]
Sorry until GTK support this, it'll not be possible.
